src/org/labkey/test/tests/InlineImagesListTest.java

```diff
@@ -52,6 +57,9 @@
 @BaseWebDriverTest.ClassTimeout(minutes = 5)
 public class InlineImagesListTest extends BaseWebDriverTest
 {
+    private final static String PROJECT_NAME = "InlineImagesListTestProject";
+    private final static String IMPORT_PROJECT_NAME = "InlineImagesListImportTestProject";
```

Comment:
For good measure, might want to change these to have spaces between words as that is more realistic.